sdks/python/apache_beam/transforms/core.py

```diff
@@ -2746,6 +2747,11 @@ def expand(self, pcoll):
     combine_fn = self._combine_fn
     fanout_fn = self._fanout_fn
 
+    if self.isinstance(pcoll.windowfn, SlidingWindows):
+      raise ValueError(
+          'CombinePerKey.with_hot_key_fanout does not yet work properly with SlidingWindows'
```

Comment:
Thanks.

please reference: https://github.com/apache/beam/issues/20528.

I recommend you import this change internally first before cutting the release candidate.